src/screens/Profile/EditProfileScreen.js

```diff
@@ -1,72 +1,119 @@
 import React from 'react';
```

Comment:
I'm ok with having the screen like this right now, but I think we should separate the functionality of changing your full name, display name, and password because it's not really intuitive. Let's figure this out in a later pr.